### PR TITLE
add readme, index.js,package.json,test.txt,fileHandler.js,.gitignore

### PR DESCRIPTION
**Pull request by chawlapalak**
add readme, index.js, package.json ,test.txt, fileHandler.js,. gitignore

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,14 @@
-# DPS909_Open_Source
-This repo is for Open source project in BSD.
+# URL-Inspector
+This is a command line interface tool to check different URLs and detect if they are good or bad or unknow.
```
Comment (mshinh) [inline README.md]: Spelling error - 'unknown'

File: README.md [hunk 2]
```diff
@@ -1,2 +1,14 @@
...
+
+# Instalation
+npm i g https://github.com/chawlapalak/Url-Inspector
```
Comment (mshinh) [inline README.md]: missing -g

File: README.md [hunk 3]
```diff
@@ -1,2 +1,14 @@
-# DPS909_Open_Source
-This repo is for Open source project in BSD.
+# URL-Inspector
+This is a command line interface tool to check different URLs and detect if they are good or bad or unknow.
+
+# Instalation
```
Comment (mshinh) [inline README.md]: Spelling error - 'installation'

File: README.md [hunk 4]
```diff
@@ -1,2 +1,14 @@
...
+npm i g https://github.com/chawlapalak/Url-Inspector
+
+# Features
+
+- You can check the different URLs by runing this script if they are working or not : url-inspector -f test.txt.
```
Comment (mshinh) [inline README.md]: Spelling error - running

File: README.md [hunk 5]
```diff
@@ -1,2 +1,14 @@
...
+- It will display good in green , if it is working.
```
Comment (mshinh) [inline README.md]: It's more understandable if the status codes are shown. For eg, 400/404 - bad, 200 - good, other- unknown, etc.

**Review by mshinh — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (chawlapalak) [reply README.md]: Fixed. Thanks.

**Pull request merged**